src/@components/homePage/card/tags.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import { NewTagIcon } from "../../../assets";
+import { BestTagIcon } from "../../../assets";
+import { RecommendTagIcon } from "../../../assets";
+import { SoldOutTagIcon } from "../../../assets";
+
+interface TagsProps {
+  value: Array<string>;
+}
+
+export default function Tags(props: TagsProps) {
+  const { value } = props;
+  function isNew() {
+    const foundtag = value.includes("new");
+    return foundtag;
+  }
+  function isBest() {
+    const foundtag = value.includes("best");
+    return foundtag;
+  }
+  function isRecommend() {
+    const foundtag = value.includes("recommend");
+    return foundtag;
+  }
+  function isSoldOut() {
+    const foundtag = value.includes("soldout");
+    return foundtag;
+  }
+  return (
+    <TagsWrapper>
+      {isNew() ? <NewTagIcon /> : null}
```

Comment:
요고두! 
```suggestion
      {isNew() && <NewTagIcon /> }
```
이렇게 쓸 수 있어요!